mne/viz/evoked.py

```diff
@@ -1941,7 +1941,7 @@ def _plot_compare_evokeds(ax, data_dict, conditions, times, do_ci, ci_dict,
 def plot_compare_evokeds(evokeds, picks=None, gfp=False, colors=None,
                          linestyles=['-'], styles=None, cmap=None,
                          vlines="auto", ci=0.95, truncate_yaxis="max_ticks",
-                         truncate_xaxis=True, ylim=dict(), invert_y=False,
+                         truncate_xaxis=True, ylim=None, invert_y=False,
```

Comment:
function calls in function / method signatures are dispreferred.  Plus, you already have a block below that converts it to `dict()` if it's `None`